lib/actions/addClass.js

```diff
@@ -0,0 +1,30 @@
+class Add {
+  constructor(){
+    this.success = false;
+  }
+
+  validator(){}
+
+  async execute(arg, project, db = {store(){}}){ 
+    this.success = false;
+    project.links[arg[0]] = arg[1];
+    try {
+      this.success = true;
+      await db.store(project)
+    } catch(err) {
+        this.success = false;
+        console.log(err);
+    }
+  }
+}
+   
+
+module.exports = { Add };
```

Comment:
I would use a default export here, instead of a named export here, but there is a some controversy about that. Some people recommend only doing named exports like you've done here. This might be something to ask some other developers about. In the end, I'll be ok with either one as long as we are consistent. Just adding these comments for something to think about and explore.

Reply:
Whats all the commotion? hahaha
Alrighty cool, default exports moving forward.